packages/fuels-core/src/types/wrappers/transaction.rs

```diff
@@ -483,6 +508,51 @@ impl CreateTransaction {
     }
 }
 
+impl GasValidation for CreateTransaction {
+    fn validate_gas(&self, min_gas_price: u64, _gas_used: u64) -> Result<()> {
```

Comment:
Good opportunity to mention that `CreateTransaction` doesn't consume gas:

```suggestion
    // We're not using `gas_used` in this implementation because `CreateTransaction` consumes no gas.
    fn validate_gas(&self, min_gas_price: u64, _gas_used: u64) -> Result<()> {
```